main.go

```diff
@@ -296,7 +296,6 @@ func main() {
 	for i := range addresses {
 		address := addresses[i]
 
-		// TODO: expose smtpd config options (timeouts, message size, and recipients)
```

Comment:
It does seem like we might want to make some of these things configurable? Might not be worth doing it right now but perhaps we shouldn't delete the TODO?

Reply:
Actually, I think this could be really useful to do now. Then if we run into a max recipient issue later on or a timeout, we can easily just modify [our config](https://github.com/grafana/deployment_tools/blob/ba0f71fab1edb8bc906138172a4ba21f747e1db2/ksonnet/lib/hosted-grafana/smtprelay.libsonnet), rather than recreating a binary and redeploying. I updated it to flags.